libs/frontend/feature-render-builder/src/components/LinkTooltip.vue

```diff
@@ -1,18 +1,23 @@
 <template>
-  <div class="link" @mousedown.stop>
+  <div ref="tooltipRef" class="link-tooltip" :style="tooltipStyle" @mousedown.stop>
     <PSInput
       v-if="editing"
       ref="linkInput"
       v-model="editedLink"
       class="link-input"
+      :datalistId="componentId"
+      :datalist="linkDatalist"
       :draggable="true"
       @dragstart.stop.prevent
       @keyup.enter="updateLink"
+      @keyup.esc="$event.srcElement.blur()"
     />
     <div v-else-if="internalLink" class="link-view" @click="gotoPage">
       {{ link }}
     </div>
-    <a v-else-if="link" :href="link" target="_blank" class="link-view"> {{ link }}</a>
+    <a v-else-if="link" :href="link" target="_blank" class="link-view" @click.stop>
```

Comment:
@abemscac I was able to get rid of the workaround by adding `@click.stop` here. I think there's something overriding the click behavior, so this makes sure the link works.